app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -87,11 +87,14 @@ class PasswordFragment : Fragment(R.layout.password_recycler_view) {
                 requireStore().refreshPasswordList()
                 binding.swipeRefresher.isRefreshing = false
             } else if (!PasswordRepository.isGitRepo()) {
-                Snackbar.make(binding.root, getString(R.string.clone_git_repo), Snackbar.LENGTH_INDEFINITE)
-                    .setAction(R.string.clone_button) {
+                BasicBottomSheet.Builder(binding.root.context)
+                    .setTitle("")
```

Comment:
Rather than these empty setTitle calls I guess we can simply relax the checks in BasicBottomSheet.Builder to check for either rather than both.

Reply:
sure @msfjarvis, it's done.